spec/services/declarations/create_spec.rb

```diff
@@ -361,20 +404,5 @@
         expect(declaration.superseded_by).to eq(original_declaration)
       end
     end
-
-    context "when the declaration_type is not valid" do
-      subject(:service) { described_class.new(**params) }
-
-      let(:declaration_type) { "started-1" }
-
-      it "returns false" do
-        expect(service.create_declaration).to be false
-      end
-
-      it "has an error" do
-        service.create_declaration
-        expect(service).to have_error(:declaration_type, :inclusion, "The entered '#/declaration_type' is not recognised.")
-      end
-    end
```

Comment:
Whats the reason for dropping this spec?

Reply:
that spec just moved - into the _validations_ section - line 294 above.